### PR TITLE
Sync with apt before installing stuffs

### PR DESCRIPTION
**Pull request by JesterOrNot**
Why don't we sync package lists before install see 
 [the docker docs](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get)

### CODE REVIEW

File: full/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 FROM buildpack-deps:disco
 
 ### base ###
-RUN yes | unminimize \
+RUN apt-get update -yqq \
```
Comment (jankeromnes) [inline full/Dockerfile]: On a side note, I think you don't need `-y` on `apt-get update` (there is no confirmation prompt).

```suggestion
RUN apt-get update -qq \
```

File: full/Dockerfile [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+    && yes | unminimize \
```
Comment (jankeromnes) [inline full/Dockerfile]: Also, do we actually need the `yes | unminimize` thing if we use `-y` with `apt-get install`? Please try removing it:

```suggestion
```

**Comment by jankeromnes** [comment]
Also, the CircleCI build of `workspace-full-vnc` failed with:

```
d82f55761eb6: Pushing  628.5MB

d82f55761eb6: Pushing  630.7MB

d82f55761eb6: Pushing  630.8MB

d82f55761eb6: Pushed 
branch-sh-sync: digest: sha256:e0923e00122cbea737fa62a2b384eec2bff1b81c6b4041b653338f3702224a7c size: 8519

Too long with no output (exceeded 10m0s): context deadline exceeded
```

Please bump the timeout for this image. 🙂

File: .circleci/config.yml [hunk 3]
```diff
@@ -39,6 +39,7 @@ jobs:
             docker load -i workspace-full.tar
             ./.circleci/build_image.sh full-vnc/Dockerfile gitpod/workspace-full-vnc
             docker save gitpod/workspace-full-vnc -o workspace-full-vnc.tar
+          no_output_timeout: 15m
```
Comment (jankeromnes) [inline .circleci/config.yml]: Please try to pick values that have already been used before

**Comment by jankeromnes** [comment]
Go installation broke again (seems unrelated to your PR though):

```
              honnef.co/go/tools/staticheck
              golang.org/x/tools/internal/telemetry/export/ocagent
              golang.org/x/tools/internal/telemetry/export/prometheus
              golang.org/x/tools/internal/lsp/debug
              golang.org/x/tools/internal/lsp/mod
              golang.org/x/tools/internal/lsp
              golang.org/x/tools/gopls/internal/hooks
              # golang.org/x/tools/gopls/internal/hooks
              go-packages/pkg/mod/golang.org/x/tools/gopls@v0.3.4/internal/hooks/analysis.go:17:30: cannot use a (type *analysis.Analyzer) as type source.Analyzer in assignment
              go-packages/pkg/mod/golang.org/x/tools/gopls@v0.3.4/internal/hooks/analysis.go:24:30: cannot use a (type *analysis.Analyzer) as type source.Analyzer in assignment
              go-packages/pkg/mod/golang.org/x/tools/gopls@v0.3.4/internal/hooks/analysis.go:27:30: cannot use a (type *analysis.Analyzer) as type source.Analyzer in assignment
              golang.org/x/tools/internal/lsp/browser
              golang.org/x/sync/errgroup
              golang.org/x/tools/internal/memoize
              golang.org/x/tools/internal/lsp/cache
              golang.org/x/tools/internal/tool
              golang.org/x/tools/internal/lsp/lsprpc
              golang.org/x/tools/internal/lsp/cmd
              build done                                  
```

**Comment by aledbf** [comment]
Closing. Invalid since the new `install-packages` script.

**Pull request closed without merging**